<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableTime">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>40</y>
      <width>891</width>
      <height>670</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777211</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>301</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string> Работа с датами:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="Date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Введите дату вида 00.00.0001:</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>261</width>
       <height>155</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="ButtonNextDay">
        <property name="text">
         <string>Узнать следующий день</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonPreviousDay">
        <property name="text">
         <string>Узнать предыдущий день</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonDuration">
        <property name="text">
         <string>Промежуток до текущей даты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonLeapYear">
        <property name="text">
         <string>Високосный год</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="Birthday">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ваш День Рождения:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineBirthday">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonBirthDaySearch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>261</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Узнать через сколько ДР</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>281</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Работа с таблицей:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonWeek">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>261</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>Узнать номер текущей недели</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="loadDateButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>730</y>
      <width>891</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить список дат</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAddDate">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>261</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить дату</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineAddTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>291</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1279</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
